admin_guide/install/manual_install.adoc

```diff
@@ -99,32 +83,7 @@ Install the packages for Ansible:
 ----
 ====
 
-== Generating and Distributing SSH Keys
-Ansible requires SSH key distribution. First, generate an SSH key on your
-master, where we will run Ansible:
-
-====
-----
-# ssh-keygen
-----
-====
-
-Do *not* use a password.
-
-An easy way to distribute your SSH keys is by using a `bash` loop:
-
-====
-[options="nowrap"]
-----
-# for host in ose3-master.example.com \
-    ose3-node1.example.com \
-    ose3-node2.example.com; \
-    do ssh-copy-id -i ~/.ssh/id_rsa.pub $host; \
-    done
-----
-====
-
-Modify the host names in the above command accordingly per your configuration.
+endif::[]
 
 == Cloning the Ansible Repository
 The configuration files for the Ansible installer are currently available
```

Comment:
These instructions still show cloning the beta branch. What should they be updated to? For both Origin and Enterprise.